### DISCUSSION TITLE
PLS HELP ME!

### DISCUSSION THREAD

**Discussion by Thegamerprogrammer** [Sid72020123/scratchconnect#43] [General]
**I Recently checked your scratchconnect forum on scratch: https://scratch.mit.edu/discuss/topic/553038/?page=1
but The Cloud Events code dosn't seem to work**

_Output:_ 
```
Traceback (most recent call last):
  File "C:\Users\HP\Desktop\Cloud_Monitor.py", line 5, in <module>
    event = variables.start_event(
AttributeError: 'CloudConnection' object has no attribute 'start_event'``
```

**Comment by Sid72020123** [comment]
Can you tell me which ScratchConnect version are you using? It gets printed when you import it.
Also, the code on that topic is old and is not updated. See the latest docs for Cloud Events: https://github.com/Sid72020123/scratchconnect#cloud-events

**Reply by Thegamerprogrammer** [reply]
I am using scratchconnect v3.4 i really want to make a cloud monitor

**Reply by Thegamerprogrammer** [reply]
ah nvm i figured it out

**Reply by Sid72020123** [reply]
Cool! So vote my answer so that it can be helpful to others :)
